### PR TITLE
Redoing safety moth

### PR DESCRIPTION
**Pull request by Horatio22**
## About The Pull Request

Making **https://github.com/fulpstation/fulpstation/pull/52** make sense with the current dept ERT files for organization reasons. It's functionally the same as the other one but less messy (hopefully)

## Why It's Good For The Game

Looks nicer than a safety moth specific file with everything in the same dm

## Changelog
:cl: Adds safety moth ERT
add: Added new things

/:cl:

### CODE REVIEW

File: fulp_modules/departmental_ert/code/safety_moth.dm [hunk 1]
```diff
@@ -0,0 +1,48 @@
+
+
+
+/datum/antagonist/ert/safety_moth/on_gain()
```
Comment (JohnFulpWillard) [inline fulp_modules/departmental_ert/code/safety_moth.dm:4]: You're doing this proc twice, once here and once in code/procs.dm

File: fulp_modules/departmental_ert/code/safety_moth.dm [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	equip_official()
+	ADD_TRAIT(owner.current, TRAIT_PACIFISM, JOB_TRAIT)
+
+/datum/antagonist/ert/safety_moth/greet()
+
+	to_chat(owner, "<B><font size=3 color=green>You are the [name].</font></B>")
+	to_chat(owner, "You are being sent on a mission to [station_name()] by Nanotrasen's Operational Safety Department. Ensure the crew is following all proper safety protocols. Board the shuttle when your team is ready.")
+
+
+/datum/antagonist/ert/safety_moth/proc/equip_official()
```
Comment (JohnFulpWillard) [inline fulp_modules/departmental_ert/code/safety_moth.dm:17]: This one, too

**Review by JohnFulpWillard — CHANGES_REQUESTED** [review]

File: fulp_modules/departmental_ert/code/safety_moth.dm [hunk 3]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (JohnFulpWillard) [inline fulp_modules/departmental_ert/code/safety_moth.dm:1]: This file isnt in the tgstation.dme

File: fulp_modules/departmental_ert/code/hardsuits.dm [hunk 4]
```diff
@@ -117,3 +117,30 @@
...
+
+// safety moth
+
+/obj/item/clothing/suit/space/eva/safety_moth
+	name = "Safety Moth safety vest"
+	desc = "The standard issue safety vest of the Safety Moth. Functions like a hardsuit and offers superb protection against environmental hazards."
+	icon_state = "hazard"
+	inhand_icon_state = "hazard"
+	flags_inv = null
+	armor = list(MELEE = 65, BULLET = 50, LASER = 50, ENERGY = 60, BOMB = 50, BIO = 100, RAD = 100, FIRE = 80, ACID = 80)
```
Comment (ATH1909) [inline fulp_modules/departmental_ert/code/hardsuits.dm:129]: mmmm, gamer armor

File: fulp_modules/departmental_ert/code/safety_moth.dm [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+/obj/item/clothing/suit/space/eva/safety_moth
+	name = "Safety Moth safety vest"
+	desc = "The standard issue safety vest of the Safety Moth. Functions like a hardsuit and offers superb protection against environmental hazards."
+	icon_state = "hazard"
+	inhand_icon_state = "hazard"
+	flags_inv = null
+	armor = list(MELEE = 65, BULLET = 50, LASER = 50, ENERGY = 60, BOMB = 50, BIO = 100, RAD = 100, FIRE = 80, ACID = 80)
+	resistance_flags = FIRE_PROOF
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	slowdown = 0
```
Comment (ATH1909) [inline fulp_modules/departmental_ert/code/safety_moth.dm:47]: these as well

File: fulp_modules/departmental_ert/code/hardsuits.dm (hunk 4 above)
Comment (Horatio22) [reply fulp_modules/departmental_ert/code/hardsuits.dm:129]: Yup, same stats as the engie ERT hardsuit

**Pull request merged**